src/cli/Vaults.ts

```diff
@@ -29,7 +30,36 @@ function makeListVaultsCommand() {
     );
 }
 
-function makeAddVaultCommand() {
+function makeScanVaultsCommand() {
+  return new commander.Command('scan')
+    .description('scan a known peer for accessible vaults')
```

Comment:
Ah here we have the `scan` command.

That means our usage of `ping` is different from network `ping`. Our `ping` is a sonar ping as it's about finding all peers currently listening on the network. It's an advertisement.

I wonder if that's a bit confusing, and we might later call that something different. The GUI obviously can communicate the sonar analogy easily, but the CLI is a bit more difficult with visual metaphors.